### DISCUSSION TITLE
ModuleNotFoundError: No module named 'ray'

### DISCUSSION THREAD

**Discussion by SMAntony** [huggingface/text-generation-inference#2573] [General]
![image](https://github.com/user-attachments/assets/b4b85065-7c86-472f-8b1b-92f1601dbd8c)

I got `ray` not found error when running TGI for meta-llama 3.1 8b model. Is this a bug or am I missing something?

**Comment by OlivierDehaene** [comment]
This warning is emitted by a dependency and can be safely ignored.
What version of TGI is it?
